### ISSUE TITLE
CSI tests are explicitly setting pod node name, which bypasses the scheduler

### ISSUE THREAD

**Issue by msau42** [kubernetes/kubernetes#70259]
**What happened**:
CSI tests are setting node name in the test pods [here](https://sourcegraph.com/github.com/kubernetes/kubernetes/-/blob/test/e2e/storage/csi_volumes.go#L83).  This is only needed for hostpath tests, but not for most other drivers.  This requirement should be pulled out into some sort of plugin config.

@sig-storage-bugs

/king bug

**Comment by msau42** [comment]
@mkimuram does your PR to merge in-tree + csi tests address this?

@kubernetes/sig-storage-bugs

**Comment by mkimuram** [comment]
> does your PR to merge in-tree + csi tests address this?

Yes. Once https://github.com/kubernetes/kubernetes/pull/68025 is merged, drivers will be able to choose to set or not to set node name.
